diff --git a/node_modules/@types/eslint/index.d.ts b/node_modules/@types/eslint/index.d.ts
index f0aa090..4c584e3 100644
--- a/node_modules/@types/eslint/index.d.ts
+++ b/node_modules/@types/eslint/index.d.ts
@@ -257,7 +257,7 @@ export namespace Rule {
         BreakStatement?: (node: ESTree.BreakStatement & NodeParentExtension) => void;
         CallExpression?: (node: ESTree.CallExpression & NodeParentExtension) => void;
         CatchClause?: (node: ESTree.CatchClause & NodeParentExtension) => void;
-        ChainExpression?: (node: ESTree.ChainExpression & NodeParentExtension) => void;
+        ChainExpression?: (node: /* ESTree.ChainExpression */ & NodeParentExtension) => void;
         ClassBody?: (node: ESTree.ClassBody & NodeParentExtension) => void;
         ClassDeclaration?: (node: ESTree.ClassDeclaration & NodeParentExtension) => void;
         ClassExpression?: (node: ESTree.ClassExpression & NodeParentExtension) => void;
